tools/tpm2_activatecredential.c

```diff
@@ -259,7 +259,7 @@ bool tpm2_tool_onstart(tpm2_options **opts) {
          {"context",        required_argument, NULL, 'c'},
          {"key-context",    required_argument, NULL, 'C'},
          {"auth-key",       required_argument, NULL, 'P'},
-         {"endorse-passwd", required_argument, NULL, 'E'},
```

Comment:
I think you can leave the existing line and it will allow multiple long options. We cant nuke this on a stable branch.